### DISCUSSION TITLE
a

### DISCUSSION THREAD

**Discussion by hell001** [onevcat/Kingfisher#1908] [Q&A]
a

**Comment by onevcat** [comment]
Sounds like a certain method of [`ImageDownloaderDelegate`](https://github.com/onevcat/Kingfisher/blob/master/Sources/Networking/ImageDownloaderDelegate.swift) is what you need? Maybe [this](https://github.com/onevcat/Kingfisher/blob/master/Sources/Networking/ImageDownloaderDelegate.swift#L71) or [this](https://github.com/onevcat/Kingfisher/blob/master/Sources/Networking/ImageDownloaderDelegate.swift#L101), depends on what exactly you want to do.

A basic use can be [found here](https://github.com/onevcat/Kingfisher/wiki/Cheat-Sheet-v4#check-http-status-code-and-determine-whether-it-is-valid).
